code/lib/core-common/vitest.d.ts

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable @typescript-eslint/no-empty-interface */
+import 'vitest';
+
+interface CustomMatchers<R = unknown> {
+  toMatchPaths(paths: string[]): R;
```

Comment:
The `expect.extend` should probably move to a test-setup file so it's available for every test, if we're adding the types like this.  It would be nice if there was a way to handle the types for an extends in just one test, though.

Reply:
sounds good to me!